### PR TITLE
Update gender rules for 1010CG schema

### PR DESCRIPTION
**Pull request by LinnJS**
# New schema
Stakeholders request for gender to not be required and "unknown" removed from the schema 

## Pull Requests to update the schema in related repositories

vets-website PR: https://github.com/department-of-veterans-affairs/vets-website/pull/13985
vets-api PR: TBA

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "6.7.4",
```
Comment (LinnJS) [inline package.json:3]: Bumped major since removing unknown is a breaking change

**Comment by LinnJS** [comment]
This PR is waiting on BE breaking changes to be updated

File: dist/21-526EZ-ALLCLAIMS-schema.json [hunk 2]
```diff
@@ -318,7 +318,7 @@
         "MI",
         "MN",
         "MS",
-        "AK",
+        "MO",
```
Comment (kevinmirc) [inline dist/21-526EZ-ALLCLAIMS-schema.json:321]: We are going to only want our 10-10CG changes in this PR.

If these changes were created when you ran build, let's put this in a separate PR. Probably will need to make a branch off of master, run the build, commit the changes, merge that PR before this one.

File: src/examples/10-10CG/example.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ const example = {
     fullName: { first: 'John Doe', middle: 'A', last: 'Doe' },
     ssnOrTin: '787787854',
     dateOfBirth: '1989-11-06',
-    gender: 'U',
+    gender: 'F',
```
Comment (kevinmirc) [inline src/examples/10-10CG/example.js:33]: This example should only have required properties defined. It's used to test the most minimal input a user can provide. So we should remove the gender value/property. This way we know dependent libraries (i.e. vets-api) can function without a gender provided.

File: test/schemas/10-10CG/schema.spec.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ import schema from '../../../src/schemas/10-10CG/schema';
 
 const testData = {
   gender: {
-    valid: ['M', 'F', 'U'],
+    valid: ['M', 'F'],
     invalid: ['A', 'BB', 'CCC'],
```
Comment (kevinmirc) [inline test/schemas/10-10CG/schema.spec.js:10]: I'd add "U" to the list of invalid examples. That was we know "U" is no longer supported and it should not be accepted.

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "6.8.4",
```
Comment (kevinmirc) [inline package.json:3]: The new version should be `7.0.0` since it's a breaking change and require a **major** version change.

https://semver.org/#summary

File: dist/21-526EZ-ALLCLAIMS-schema.json (hunk 2 above)
Comment (LinnJS) [reply dist/21-526EZ-ALLCLAIMS-schema.json:321]: Opened another PR to isolate changes:
https://github.com/department-of-veterans-affairs/vets-json-schema/pull/487

These updates occurred due to me just running `yarn build`

File: src/examples/10-10CG/example.js (hunk 3 above)
Comment (LinnJS) [reply src/examples/10-10CG/example.js]: Removed all instances of `gender` from this example

File: test/schemas/10-10CG/schema.spec.js (hunk 4 above)
Comment (LinnJS) [reply test/schemas/10-10CG/schema.spec.js]: Added "U" to `invalid` case

File: package.json (hunk 5 above)
Comment (LinnJS) [reply package.json]: Bumped to 7.0.0

**Comment by kevinmirc** [comment]
@LinnJS Backend is ready for these changes. We can merge.

https://github.com/department-of-veterans-affairs/vets-api/pull/4878

**Comment by kevinmirc** [comment]
Closing this ticket in favor of the same changes made in this PR: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/502

Uses same commits but diff it clean.
